### PR TITLE
hash in passwords

### PR DESCRIPTION
**Pull request by sveip**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/configuration/secrets.markdown [hunk 1]
```diff
@@ -35,6 +35,7 @@ The `secrets.yaml` file contains the corresponding password assigned to the iden
 ```yaml
 http_password: YOUR_PASSWORD
 ```
+If password contains the hash character, you have to embed it in double quotes to avoid it being treated as a comment. `http_password: "YOUR_PASSWORD_WITH_#"
```
Comment (fabaff) [inline source/_docs/configuration/secrets.markdown:38]: That's YAML and already covered in the documentation.

Comment (sveip) [reply source/_docs/configuration/secrets.markdown:38]: Well, if that logic is applied generally, most of the doc would be superfluous ;) It's common mistake, and not mentioned anywhere in the docs that are usually read y inexperienced users. Maybe it should rather be mentined as a "Tip"?

Comment (fabaff) [reply source/_docs/configuration/secrets.markdown:38]: Well, moving the secrets out the configuration is an advanced topic and placed in that section while the YAML intro is the very first topic when it comes to configuration.

Comment (sveip) [reply source/_docs/configuration/secrets.markdown:38]: Ah, sorry, then I misunderstood your first comment. If it's covered in the config then it's probably OK. Even though I don't think it's bad to ad hints and tricks, it's hard to "overdocument."

**Pull request closed without merging**